front/src/applications/rollingStockEditor/components/RollingStockEditorForm.tsx

```diff
@@ -420,25 +385,72 @@ const RollingStockEditorForm = ({
     );
   };
 
-  // TODO: use InputSNCF and InputGroupSNCF
+  useEffect(() => {
+    setIsValid(true);
+    RollingStockschemaProperties.forEach((property) => {
+      if (
+        (property.title === Parameter[property.title as Parameter] &&
+          Number.isNaN(rollingStockValues[property.title])) ||
+        (rollingStockValues[property.title] as number) < (property.min as number) ||
+        (rollingStockValues[property.title] as number) > (property.max as number)
```

Comment:
I don't really understand this test

Reply:
if the input is empty
or if the number is less than the minimum
or if the number is more than the maximum